libfastpm/pngaussian.c

```diff
@@ -79,12 +101,31 @@ fastpm_png_transform_potential(PM * pm, FastPMFloat * g_x, FastPMPNGaussian * pn
     /*FIXME: transform the potential !*/
     ptrdiff_t i;
     double avg_g_squared = 0.0;
+    /* MS: Should we better do this globally over all processors? */
     for(i = 0; i < pm_size(pm); i ++) {
 	avg_g_squared += g_x[i] * g_x[i];
     }
+    avg_g_squared /= (double) pm_size(pm);
```

Comment:
There is also volume / resolution dependency. If you change the box size to 1024, The difference can go up to ~10x.